server/src/features/products/model.ts

```diff
@@ -1,23 +1,38 @@
-import { PrismaClient, Product } from "@prisma/client";
+import { Category, Prisma, PrismaClient, Product } from "@prisma/client";
+import { stripe } from "../orders/controller";
 const prisma = new PrismaClient();
 
 export type ProductCreationParams = Pick<
   Product,
-  "name" | "description" | "price" | "available" | "categoryId"
+  "name" | "description" | "price" | "available" | "categoryName"
 >;
 
 export class ProductModel {
-  async getAll(): Promise<Product[]> {
-    return await prisma.product.findMany();
-  }
   async create(data: ProductCreationParams): Promise<Product> {
-      return await prisma.product.create({
-        data: {
-          ...data,
-        },
-      });
+    const product = await stripe.products.create({
+      name: data.name,
+      default_price_data: {
+        unit_amount: Number(data.price) * 100,
```

Comment:
Same as the /100, you should probably create a stripe service that does these stripes conversion for you, and also puts the payload to be sent to the api in the right form :)